### DISCUSSION TITLE
Celebrations!

### DISCUSSION THREAD

**Discussion by N3RDIUM** [N3RDIUM/zen#11] [Announcements]
![image](https://user-images.githubusercontent.com/74598401/135040774-caf95e55-b70e-4b78-9909-94fb91a0ea98.png)

#### We have made a new release with a lot of bug fixes and new features!
@Super45coder @Sreehari521 @e-coders 
#### (I cannot mention genius398 as he is busy with his exams)

**Comment by Super45coder** [comment]
HOORAY! I am planning to make a storage called Skydrive. Can you help me?
________________________________
From: somePythonProgrammer ***@***.***>
Sent: Thursday, September 30, 2021 9:53 AM
To: somePythonProgrammer/JARVIS ***@***.***>
Cc: Goradia, Aarush H ***@***.***>; Mention ***@***.***>
Subject: [somePythonProgrammer/JARVIS] Celebrations! (Discussion #11)

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


[image]<https://user-images.githubusercontent.com/74598401/135040774-caf95e55-b70e-4b78-9909-94fb91a0ea98.png>

CELEBRATIONS!
We have made a new release with a lot of bug fixes and new features!

@Super45coder<https://github.com/Super45coder> @Sreehari521<https://github.com/Sreehari521> @e-coders<https://github.com/e-coders>

(I cannot mention genius398 as he is busy with his exams)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/somePythonProgrammer/JARVIS/discussions/11>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATDMIRTLXY3ETCAXQY6A6GLUESBWTANCNFSM5FCVGUIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by N3RDIUM** [reply]
Sure!

**Comment by devashish2024** [comment]
YAY!!!

**Comment by Super45coder** [comment]
I will start coding it tomorrow and give you access to it.
________________________________

**Reply by N3RDIUM** [reply]
Sure!

**Reply by N3RDIUM** [reply]
@Super45coder What platform are you going to use?

**Comment by Super45coder** [comment]
Github
________________________________

**Reply by N3RDIUM** [reply]
@Super45coder  I mean which language?

**Comment by Super45coder** [comment]
JS
________________________________

**Reply by N3RDIUM** [reply]
OK

**Comment by Super45coder** [comment]
Also, check out my project on scratch:
https://scratch.mit.edu/projects/571835145/
________________________________
